packages/rolldown/src/cli/commands/help.ts

```diff
@@ -0,0 +1,39 @@
+import { logger } from '../utils'
+import {
+  version,
+  description,
+} from '../../../package.json' assert { type: 'json' }
+import { CLI_OPTIONS } from '../options'
+import { bold, cyan, underline } from '../colors'
+
+const HELP_TEMPLATE = `rolldown version ${version}
+
+${description}
+=====================================
+
+${bold(underline('USAGE'))} rolldown [OPTIONS]
+
+${bold(underline('OPTIONS'))}
+__OPTIONS__
+`
+
+export function showHelp() {
+  logger.log(
+    HELP_TEMPLATE.replace(
+      '__OPTIONS__',
+      Object.entries(CLI_OPTIONS)
+        .map(([option, { short, hint, description }]) => {
+          let optionStr = ''
+          if (short) {
+            optionStr += `-${short}, `
+          }
+          optionStr += `--${option} `
+          if (hint) {
+            optionStr += `<${hint}> `
+          }
+          return cyan(optionStr.padEnd(30)) + description
+        })
+        .join('\n'),
```

Comment:
Can you add an indentation for each `option` ?

Reply:
![image](https://github.com/user-attachments/assets/d706c665-5e4e-4da3-8238-d3d23f7f6aaa)

Also we should fix the wrong indentation before

Reply:
Done, two spaces are added to each option